### DISCUSSION TITLE
Wrong Date/TIme values stored when using bind of Form Date Picker

### DISCUSSION THREAD

**Discussion by sergey-dme** [Budibase/budibase#2537] [Help] [answered]
Hello, 

Just noticed that a ticket I raised was closed today while the issue is still present and unsolved.
https://github.com/Budibase/budibase/issues/2108 Details and reproduction steps described in last two messages.

Once user added a new record/row in my app, I still need to manually correct date/time values in builder edit records for every new created record, otherwise sorting order and formulas in filters do not work. 

Currently sorting order, formulas in filters do not work correctly for a Date/Time field, WHEN some rows in the table have values received from bind of a date time picker, some of them values directly from Form date picker component, the others were stored using `{{ now }}` binding

**Comment by sergey-dme** [comment]
Just to show the case:
1. Creating new record with two date/time fields, **DateA** and **DateB**

<img width="398" alt="Screenshot 2021-09-03 at 21 22 39" src="https://user-images.githubusercontent.com/53024832/132050867-6ba32e07-89fc-4a5f-95f2-c57de24ce2ef.png">

2. Check the record created

<img width="720" alt="Screenshot 2021-09-03 at 21 23 31" src="https://user-images.githubusercontent.com/53024832/132050925-0859a72a-40e0-4e76-9df0-819ab505c451.png">

3. Open it to edit. Now we can see that wrong value was stored for **DateB**:

<img width="676" alt="Screenshot 2021-09-03 at 21 23 41" src="https://user-images.githubusercontent.com/53024832/132050964-c0b31295-f9bb-48c8-824e-87d5954a23d4.png">


Please help

**Comment by mjashanks** [comment, marked as answer]
Hey @sergey-dme - I confirm that this is still an issue, have re-opened #2108. Thanks!
